WebDriverAgentLib/Commands/FBElementCommands.m

```diff
@@ -116,11 +117,17 @@ + (NSArray *)routes
 + (id<FBResponsePayload>)handleGetAttribute:(FBRouteRequest *)request
 {
   FBElementCache *elementCache = request.session.elementCache;
```

Comment:
Probably a test started failing after this change. (The test passed with 2.28.0)

```
    def test_immediate_value
      w3c_scroll @@driver

      @@core.wait { @@driver.find_element :accessibility_id, 'Text Fields' }.click

      text = @@core.wait { @@driver.find_element :class, 'XCUIElementTypeTextField' }
      text.immediate_value 'hello'

      text = @@core.wait { @@driver.find_element :name, 'hello' }
      assert_equal 'hello', text.value  # Here raised StaleElementReferenceError

      @@driver.back
    end
```

https://dev.azure.com/kazucocoa/ruby_lib_core/_build/results?buildId=2084&view=logs&j=dbeb3e1e-1d01-5eac-7795-96e0486c2707&t=b8f51b39-e272-50fe-6ca1-1652821ae7c8

```
2020-10-26 16:25:21:333 - [HTTP] <-- POST /wd/hub/session/f6d2d57a-0ac8-4328-8c52-fdd0aae4f1f6/element 200 381 ms - 137
2020-10-26 16:25:21:333 - [HTTP] 
2020-10-26 16:25:21:335 - [HTTP] --> GET /wd/hub/session/f6d2d57a-0ac8-4328-8c52-fdd0aae4f1f6/element/B0000000-0000-0000-5D1E-000000000000/attribute/value
2020-10-26 16:25:21:335 - [HTTP] {}
2020-10-26 16:25:21:336 - [debug] [W3C (f6d2d57a)] Calling AppiumDriver.getAttribute() with args: ["value","B0000000-0000-0000-5D1E-000000000000","f6d2d57a-0ac8-4328-8c52-fdd0aae4f1f6"]
2020-10-26 16:25:21:336 - [debug] [XCUITest] Executing command 'getAttribute'
2020-10-26 16:25:21:336 - [debug] [WD Proxy] Matched '/element/B0000000-0000-0000-5D1E-000000000000/attribute/value' to command name 'getAttribute'
2020-10-26 16:25:21:337 - [debug] [WD Proxy] Proxying [GET /element/B0000000-0000-0000-5D1E-000000000000/attribute/value] to [GET http://127.0.0.1:8100/session/94532247-FD1C-4A08-AF15-17C18D89AD99/element/B0000000-0000-0000-5D1E-000000000000/attribute/value] with no body
2020-10-26 16:25:21:352 - [Xcode]     t =   106.35s Requesting snapshot of accessibility hierarchy for app with pid 7773
2020-10-26 16:25:21:354 - [Xcode] 
2020-10-26 16:25:21:441 - [Xcode]     t =   106.45s Find: Descendants matching type Any
2020-10-26 16:25:21:441 - [Xcode] 
2020-10-26 16:25:21:443 - [Xcode]     t =   106.45s Find: Elements matching predicate '"hello" IN identifiers'
2020-10-26 16:25:21:443 - [Xcode] 
2020-10-26 16:25:21:448 - [Xcode]     t =   106.46s Find: Identity Binding
2020-10-26 16:25:21:448 - [Xcode] 
2020-10-26 16:25:21:456 - [WD Proxy] Got response with status 404: {"value":{"error":"stale element reference","message":"The previously found element \"\"hello\" TextField\" is not present in the current view anymore. Make sure the application UI has the expected state. You could also try to switch the binding strategy using the 'boundElementsByIndex' setting for the element lookup. Original error: No matches found for Identity Binding from input {(\n    value: hello, Keyboard Focused\n)}","traceback":"(\n\t0   CoreFoundation                      0x00007fff23e3cf0e __exceptionPreprocess + 350\n\t1   libobjc.A.dylib                     0x00007fff50ba89b2 objc_exception_throw + 48\n\t2   WebDriverAgentLib                   0x00000001048d4815 -[XCUIElement(FBUtilities) fb_takeSnapshot] + 1077\n\t3   WebDriverAgentLib                   0x00000001048f3dc9 -[FBElementCache elementForUUID:resolveForAdditionalAttributes:] + 345\n\t4   WebDriverAgentLib                   0x00000001048f8e0c +[FBElementCommands handleGetAttribute:] + 668\n\t5   WebDriverAgentLib                   0...
2020-10-26 16:25:21:456 - [debug] [W3C] Matched W3C error code 'stale element reference' to StaleElementReferenceError
2020-10-26 16:25:21:458 - [debug] [W3C (f6d2d57a)] Encountered internal error running command: StaleElementReferenceError: The previously found element ""hello" TextField" is not present in the current view anymore. Make sure the application UI has the expected state. You could also try to switch the binding strategy using the 'boundElementsByIndex' setting for the element lookup. Original error: No matches found for Identity Binding from input {(
2020-10-26 16:25:21:458 - [debug] [W3C (f6d2d57a)]     value: hello, Keyboard Focused
2020-10-26 16:25:21:458 - [debug] [W3C (f6d2d57a)] )}
2020-10-26 16:25:21:460 - [debug] [W3C (f6d2d57a)]     at errorFromW3CJsonCode (/Users/runner/hostedtoolcache/node/10.22.1/x64/lib/node_modules/appium/node_modules/appium-base-driver/lib/protocol/errors.js:780:25)
2020-10-26 16:25:21:461 - [debug] [W3C (f6d2d57a)]     at ProxyRequestError.getActualError (/Users/runner/hostedtoolcache/node/10.22.1/x64/lib/node_modules/appium/node_modules/appium-base-driver/lib/protocol/errors.js:663:14)
2020-10-26 16:25:21:461 - [debug] [W3C (f6d2d57a)]     at JWProxy.command (/Users/runner/hostedtoolcache/node/10.22.1/x64/lib/node_modules/appium/node_modules/appium-base-driver/lib/jsonwp-proxy/proxy.js:273:19)
2020-10-26 16:25:21:461 - [debug] [W3C (f6d2d57a)]     at process._tickCallback (internal/process/next_tick.js:68:7)
2020-10-26 16:25:21:462 - [HTTP] <-- GET /wd/hub/session/f6d2d57a-0ac8-4328-8c52-fdd0aae4f1f6/element/B0000000-0000-0000-5D1E-000000000000/attribute/value 404 126 ms - 1415
2020-10-26 16:25:21:462 - [HTTP] 
```

<img src="https://user-images.githubusercontent.com/5511591/97204406-655fba00-17f9-11eb-8259-2d753d62ec9e.png" width=300>